### PR TITLE
docs:Updating documentation for Konko provider

### PR DESCRIPTION
**Pull request by shivanimodi16**
- **Description:** A small update to the Konko provider documentation.

### CODE REVIEW

File: docs/docs/integrations/providers/konko.mdx [hunk 1]
```diff
@@ -1,86 +1,63 @@
...
+## Chat Models
 
-## Examples of Endpoint Usage
+See a usage [example](/docs/integrations/chat/konko).
 
 
 - **ChatCompletion with Mistral-7B:**
+
   ```python
   chat_instance = ChatKonko(max_tokens=10, model = 'mistralai/mistral-7b-instruct-v0.1')
```
[the remark this answers is not in the document]
Comment (shivanimodi16) [reply docs/docs/integrations/providers/konko.mdx:60]: Thank you pointing it out! Added that

**Review by leo-gan — APPROVED** [review]
lgtm

**Review by leo-gan — APPROVED** [review]
LGTM

**Comment by shivanimodi16** [comment]
@leo-gan Thank you for a prompt response! Could we also merge this?

**Pull request merged**